composer/core/state.py

```diff
@@ -727,6 +727,10 @@ def fsdp_state_dict_type(self):
             return self.fsdp_config['state_dict_type']
         return 'full'
 
+    @property
+    def fsdp_fallback_to_monolithic(self):
```

Comment:
Is there a reason this should be public?

Reply:
how do I make it private